ls1intum/Artemis
# `General`: Do not send version control access token if not required

### Checklist
#### General




### Motivation and Context
Prevent sending unnecessary information

### Steps for Testing

Prerequisites:
- 1 Students
- 1 Programming Exercise

1. Log in to Artemis on a server with vcs access token setup
2. Make sure the user has a vcs access token
3. Navigate into the clone dialog and make sure the token is shown
4. Use the communication features and make sure in the network tab that the vcs access token is not sent with the user in the json


### Review Progress






#### Code Review
#### Manual Tests
